pytorch_lightning/plugins/precision/apex_amp.py

```diff
@@ -98,3 +98,11 @@ def state_dict(self) -> Dict[str, Any]:
 
     def load_state_dict(self, state_dict: Dict[str, Any]) -> None:
         amp.load_state_dict(state_dict)
+
+    @property
+    def amp_backend(self) -> Optional[AMPType]:
```

Comment:
I think we can remove L33?

Reply:
we can't remove it since one is `backend` and the other is `amp_backend` ?

Reply:
I didn't see anywhere using it and they two are the same thing. @daniellepintz mentioned "this is deprecated in https://github.com/PyTorchLightning/pytorch-lightning/pull/12069"